test/services/bill-runs/two-part-tariff/fetch-charge-versions.service.test.js

```diff
@@ -22,17 +22,17 @@ const WorkflowHelper = require('../../../support/helpers/workflow.helper.js')
 
 // Thing under test
 const FetchChargeVersionsService = require('../../../../app/services/bill-runs/two-part-tariff/fetch-charge-versions.service.js')
+const regionHelper = require('../../../support/helpers/region.helper.js')
```

Comment:
Think this just needs to be bumped up to the helpers section.

Reply:
I think that was caused by VSCode trying to be helpful and adding the `require` when I used a lowercase 'r' for the region helper on line 35. I've got the region helper in there twice, DOH!